src/clib/pio.h

```diff
@@ -719,7 +719,7 @@ extern "C" {
 
     /* Init decomposition with 0-based compmap array. */
     int PIOc_init_decomp(int iosysid, int basetype, int ndims, const int *dims, int maplen,
-                         const PIO_Offset *compmap, int *ioidp, const int *rearranger,
+                         const PIO_Offset *compmap, int *ioidp, int rearranger,
```

Comment:
Doesn't this make the rearranger a required argument?   If the user sets a default rearranger in PIOc_init_Int* then they shouldn't need to pass it again here.